README.md

```diff
@@ -8,61 +8,79 @@ Head over to the [marketplace](https://www.contentful.com/marketplace/) and foll
+It is the responsibility of all partner app developers to ensure their app is kept up to date with the latest versions of all dependencies required by their app. This is required not only to ensure their app continues to function as expected alongside the constantly evolving Contentful platform, but also to ensure that critical security updates are applied in a timely fashion and are not slowed down by out of date cross dependencies.
 
-To keep depedencies up to date, Contentful uses [Dependabot](https://docs.github.com/en/code-security/dependabot) in this repository. Dependabot will:
+To keep dependencies up to date, Contentful uses [Dependabot](https://docs.github.com/en/code-security/dependabot) in this repository. Dependabot will:
 
-* Open PRs to propose dependency updates for every depedency listed in your app's package.json file.
-* Automatically approve and merge PRs for patch and minor version updates.
-* Hold off on merging PRs for major version updates pending manual review by a human.
+- Open PRs to propose dependency updates for every dependency listed in your app's package.json file.
+- Automatically approve and merge PRs for patch and minor version updates.
+- Hold off on merging PRs for major version updates pending manual review by a human.
 
 > [!TIP]
 > We recommend specifying bare versions (e.g. `1.2.3`) of dependencies in your package.json file vs. a range of versions (e.g. `1.2.x` or `~1.2.3`). Dependabot will increment versions in your package.json file automatically.
 
 As an app developer, it is important to review and approve any dependency update PRs assigned to you on a frequent basis. Specifically we ask that you:
 
-* Provide a reasonably confident test suite that exercises the main functionality of your app. Test suites generally do an excellent job of catching errors caused by backward breaking compatibility changes in updated depedencies.
-* Respond to, review, and informally approve (adding a PR comment suffices as an approval) any dependency-related PRs that are opened against your app (whether by Dependabot or the Contentful team) in a timely fashion. If errors exist in aformentioned dependency-related PRs, please manually resolve these errors. 
-* Talk to us if you have a specific requirement to firmly pin a dependency version. We can make custom modifications to our Dependabot configuration to support exceptions as needed.
+- Provide a reasonably confident test suite that exercises the main functionality of your app. Test suites generally do an excellent job of catching errors caused by backward breaking compatibility changes in updated dependencies.
+- Respond to, review, and informally approve (adding a PR comment suffices as an approval) any dependency-related PRs that are opened against your app (whether by Dependabot or the Contentful team) in a timely fashion. If errors exist in aforementioned dependency-related PRs, please manually resolve these errors.
```

Comment:
Do we need "informally" here? Just seems like an opportunity for misunderstanding, especially since the parenthetical explicitly describes the expectation.